### ISSUE TITLE
names with '&' will break calculations from hyperlink

### ISSUE THREAD

**Issue by Red-Thirteen** [GustavoFernandes/order-splitter#36]
example: 

```
1 	12 Nuggets - 	$4.39
	Sauce: Ketchup
	Label for:Alice

2 	Tots - 	$1.99
	Label for:Bob

1 	6 Wings - 	$6.29
	Wing Seasoning: Sweet Honey BBQ
	Label for: Harneys & sons,
```

https://gustavofernandes.github.io/order-splitter/index.html?tax=0&fee=0&tip=20&Alice=4.39&Bob=1.99&%20Harneys%20&%20sons,=6.29

you can click on it, but it'll break the calculations for Harneys & sons

<img width="590" alt="screen shot 2017-02-25 at 7 37 06 pm" src="https://cloud.githubusercontent.com/assets/2091690/23336070/d64aaf00-fb91-11e6-89b2-e7fda980a444.png">

**Comment by GustavoFernandes** [comment]
Fixed by #38

**Comment by GustavoFernandes** [comment]
The name `Hirsh H#ir&sh%@!` doesn't encode properly for the hyperlink

**Comment by Red-Thirteen** [comment]
beat me to reopening it

**Comment by GustavoFernandes** [comment]
Fixed in v0.5.1
